Core/HLE/sceIo.cpp

```diff
@@ -66,7 +66,8 @@ static const int ERROR_ERRNO_FILE_ALREADY_EXISTS          = 0x80010011;
 static const int ERROR_MEMSTICK_DEVCTL_BAD_PARAMS         = 0x80220081;
 static const int ERROR_MEMSTICK_DEVCTL_TOO_MANY_CALLBACKS = 0x80220082;
 static const int ERROR_KERNEL_BAD_FILE_DESCRIPTOR         = 0x80020323;
-
+static const int ERROR_KERNEL_ILLEGAL_PERMISSION          = 0x800200d1;
```

Comment:
Should we use SCE_KERNEL_ERROR_ILLEGAL_PERM instead?  Same value, and used other places.

-[Unknown]